spec/lib/activitypub/activity/like_spec.rb

```diff
@@ -21,126 +21,123 @@
     subject { described_class.new(json, sender) }
 
     before do
+      stub_request(:get, 'http://example.com/emoji.png').to_return(body: attachment_fixture('emojo.png'))
       subject.perform
     end
 
     it 'creates a favourite from sender to status' do
       expect(sender.favourited?(status)).to be true
     end
-  end
-
-  describe '#maybe_process_misskey_reaction' do
-    subject { described_class.new(json, sender) }
-
-    context 'when given a valid reaction' do
-      let(:json) do
-        {
-          '@context': 'https://www.w3.org/ns/activitystreams',
-          id: 'foo',
-          type: 'Like',
-          _misskey_reaction: '👍',
-          actor: ActivityPub::TagManager.instance.uri_for(sender),
-          object: ActivityPub::TagManager.instance.uri_for(status),
-        }.with_indifferent_access
-      end
-
-      before do
-        subject.perform
-      end
 
-      it 'creates a status reaction from sender to status' do
-        expect(sender.reacted?(status, '👍')).to be true
+    context 'when a Misskey reaction' do
+      context 'with emoji' do
+        let(:json) do
+          {
+            '@context': 'https://www.w3.org/ns/activitystreams',
+            id: 'foo',
+            type: 'Like',
+            _misskey_reaction: '👍',
```

Comment:
Apparently, Misskey also sends `content` with the reacted emoji, at least according to the code, but I have no way of verifying.